example/bi-lstm-sort/sort_io.py

```diff
@@ -1,9 +1,11 @@
 # pylint: disable=C0111,too-many-arguments,too-many-instance-attributes,too-many-locals,redefined-outer-name,fixme
 # pylint: disable=superfluous-parens, no-member, invalid-name
+from __future__ import print_function
 import sys
 sys.path.insert(0, "../../python")
 import numpy as np
 import mxnet as mx
+from future.utils import viewitems
```

Comment:
`__future__` comes with python by default. future doesn't.

simply use for k, v in len_dict.items()